.github/workflows/cd.yaml

```diff
@@ -30,6 +31,11 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y --no-install-recommends libsdl2-dev libpng-dev libjpeg-turbo8-dev libopenal-dev libmad0-dev libglew-dev libgl1-mesa-dev libegl1-mesa-dev libgles2-mesa-dev uuid-dev
           python3 -m pip install --user scons
+      - name: Setup sccache
+        uses: ./.github/sccache
+        with:
+          read-only: 'true'
+          cache-key: ubuntu-20.04-desktop
```

Comment:
Can you unify the cache keys to some sensible, shared format? right now it's a mix of hardcoded strings, `{{ matrix.os }}` and `${{ matrix.os }}-desktop`

Reply:
what about `{{ runner.os }}-{{ runner.arch }}`?

As for `-desktop`, i'm fine with leaving it there but perhaps make it clearer that this is about GL being present. perhaps `-headless` for those without.

Reply:
`runner.os` doesn't include the OS version :confused: So I can't use that unfortunately.

I could rename `desktop` to `opengl`? would that help? I don't think `headless` makes sense (just like `desktop` makes no sense either).

Reply:
> `runner.os` doesn't include the OS version 😕 So I can't use that unfortunately.

why is that relevant?

> I could rename `desktop` to `opengl`? would that help? I don't think `headless` makes sense (just like `desktop` makes no sense either).

good enough

Reply:
> why is that relevant?

I guess it's only relevant for the Ubuntu jobs (20 vs 22).

Reply:
But even then - what is the difference? The only big kicker i can think of is dependency versions, and us caching the compilation of *our* code shouldn't be relevant to that.

I mean, at that point it's worth asking why we have jobs for both ubuntu versions in the first place. Unless it's explicitly to test everything on both versions, we might as well share the cache between them.

Reply:
Compiler version too. From my perspective I'm fine with just having one Ubuntu version